test/dotnet-format.Tests/tests/dotnet-format.UnitTests.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <RootNamespace>Microsoft.CodeAnalysis.Tools.Tests</RootNamespace>
+    <!-- Copy nuget assemblies to build directory so that Microsoft.CodeAnalysis.Features.* can be located when running tests. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <RollForward>LatestMajor</RollForward>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <DefaultExcludesInProjectFolder>$(DefaultExcludesInProjectFolder);binaries\**;projects\**</DefaultExcludesInProjectFolder>
+    <OutputType>Exe</OutputType>
```

Comment:
This is the unit test project of dotnet-format. 
1. It is moved here and changed the output type to exe.
2. <ProjectReference Include="..\..\Microsoft.NET.TestFramework\Microsoft.NET.TestFramework.csproj" /> has been added.
These seems like a common pattern in the repo. Let me know if it's not correct.